automation/test-linux.sh

```diff
@@ -62,7 +62,7 @@ spec:
 EOF
 
 sizes=("tiny" "small" "medium" "large")
-workloads=("desktop" "server" "highperformance")
+workloads=("desktop" "server")
```

Comment:
This will also disable `highperformance` tests for the Windows templates. Can you narrow it down to Fedora/RHEL?

Reply:
The Windows and Linux tests are located in separate files, so this should not disable the tests for Windows itself. For more details, please refer to this [line](https://github.com/kubevirt/common-templates/blob/master/automation/test-windows.sh#L10)
Thank you for your understanding regarding the high-performance issue!

Reply:
I see, sorry I missed that.